CrossPlatformDesktopProject/CrossPlatformDesktopProject/Link/LinkFacingEastState.cs

```diff
@@ -40,6 +41,7 @@ void ILinkState.setTextureIndex(int index)
         }
         void ILinkState.TakeDamage()
         {
+            player.link_health++;
```

Comment:
Similar to comment from Game1, have it be player.link_health-- instead of ++ to more accurately reflect what's happening